Testing/integration/VM/InstallationTool/BUILD

```diff
@@ -0,0 +1,44 @@
+load(
+    "@build_bazel_rules_apple//apple:macos.bzl",
+    "macos_command_line_application",
+)
+
+objc_library(
+    name = "installationtool_lib",
+    srcs = [
+        "MacOSRestoreImage.m",
+        "MacOSVirtualMachineInstaller.m",
+        "main.m",
+    ],
+    hdrs = [
+        "MacOSRestoreImage.h",
+        "MacOSVirtualMachineInstaller.h",
+    ],
+    sdk_frameworks = [
+        "Virtualization",
+    ],
+    deps = [
+        "//Testing/integration/VM/Common:configuration_helper",
+        "//Testing/integration/VM/Common:delegate",
+        "//Testing/integration/VM/Common:error",
+    ],
+)
+
+filegroup(
+    name = "entitlements",
+    srcs = [
+        "InstallationTool.entitlements",
+    ],
+)
+
+macos_command_line_application(
+    name = "InstallationTool",
+    codesignopts = [
+        "--entitlements=Testing/integration/VM/InstallationTool/InstallationTool.entitlements",
```

Comment:
Can you use the `entitlements` attribute instead? https://github.com/bazelbuild/rules_apple/blob/master/doc/rules-macos.md#macos_application-entitlements

Reply:
We can, however `entitlements` isn't supported for `macos_command_line_application` so all of these would need to use `macos_application` and have the supporting Info.plist added. Doing that still feels cleaner than this "hack" though, so will change.